types/auth0-lock/index.d.ts

```diff
@@ -223,8 +221,20 @@ interface Auth0LockStatic {
     on(event: string, callback: (...args: any[]) => void): void;
 }
 
-declare var Auth0Lock: Auth0LockStatic;
+interface Auth0LockStatic extends Auth0LockCore {
+    new (clientId: string, domain: string, options?: Auth0LockConstructorOptions): Auth0LockStatic;
+}
+
+// additional options for passwordless mode
+interface Auth0LockPasswordlessConstructorOptions extends Auth0LockConstructorOptions {
+    passwordlessMethod?: string;
+}
+
+interface Auth0LockPasswordlessStatic extends Auth0LockCore {
+    new (clientId: string, domain: string, options?: Auth0LockPasswordlessConstructorOptions): Auth0LockPasswordlessStatic;
+}
 
 declare module "auth0-lock" {
-    export default Auth0Lock;
```

Comment:
👋  Just leave the default export and add the extra exports in-order to be consistent with auth-lock libary plus that way there are no breaking changes.

https://github.com/auth0/lock
```js
import Auth0Lock from 'auth0-lock';
// OR
import { Auth0Lock } from 'auth0-lock';
import { Auth0LockPasswordless } from 'auth0-lock';
```

Reply:
I added default export